website/manual.md

```diff
@@ -300,22 +300,22 @@ const { listen, copy } = Deno;
 When this program is started, the user is prompted for permission to listen on
 the network:
 
-```shellsession
+```shell
 $ deno run https://deno.land/std/examples/echo_server.ts
 ⚠️  Deno requests network access to "listen". Grant? [a/y/n/d (a = allow always, y = allow once, n = deny once, d = deny always)]
```

Comment:
This is not valid shell script - it's showing a command and output. I suggest removing highlighting entirely.